### ISSUE TITLE
dropout effect

### ISSUE THREAD

**Issue by taki0112** [junyanz/pytorch-CycleGAN-and-pix2pix#212]
If you use dropout, How does the result come out?

better ? or not ?

**Comment by junyanz** [comment]
It achieves similar results for CycleGAN. I would recommend that you use Dropout fox pix2pix to prevent overfitting.
